src/main/scala/nl/knaw/dans/easy/export/EasyExportDataset.scala

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+  * Copyright 2015 DANS - Data Archiving and Networked Services
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  ******************************************************************************/
+
+package nl.knaw.dans.easy.export
+
+import java.io.File
+
+import nl.knaw.dans.easy.export.FOXML.strip
+import org.slf4j.LoggerFactory
+
+import scala.util.Try
+import scala.xml.Node
+
+class EasyExportDataset(s: Settings) {
```

Comment:
@DANS-KNAW/easy 
Special attention for  for educational purposes

Problem:

An implicit argument makes an actual parameter list shorter. But the formal argument list becomes more complex , the IDE highlighting facility does not show as easily where it is actually used but worst of all: it can cause runtime errors when somewhere on the stack the implicit declaration is forgotten. Compile time errors are preferable over runtime errors, the sooner you detect an error, the cheaper (in terms of man-hours) fixing it.

Solution:

I refactored the Settings from an implicit argument to an explicit constructor argument. For that purpose I had to move most object methods into a class method. The companion object now only has a main method that converts command line arguments into a Settings instance and calls the run method of a class-instance.

As a side effect of this refactoring I discovered that settings was no longer used by one of the other classes. But that is just coincidence.